README.org

```diff
@@ -257,12 +257,15 @@ their descriptions.
   =consult-project-function= returns non-nil, =consult-grep= searches the
   current project directory. Otherwise the =default-directory= is searched. If
   =consult-grep= is invoked with prefix argument =C-u M-s g=, you can specify the
-  directory manually.
+  directory manually.  With any other prefix argument, =C-u C-u M-s g=, you can
+  enter any number (including 0) space-separated directories followed by any number
+  of wildcarded paths, like "~/src/*.c" to search over.
```

Comment:
Also here, we should get rid of the restriction.

Reply:
Yes, handled.